packages/zent/src/alert/README-en_US.md

```diff
@@ -0,0 +1,159 @@
+## Alert 
+
+Alert, providing clear prompt messages.
+
+### Usage guide
+
+-  The content is as simple as possible to reduce reading barrier.
+-  No more than two buttons for Alert, keeping the logic simple.
+
+### Code demo
+
+:::demo Basic usage
+```jsx
+import { Alert } from 'zent';
+
+ReactDOM.render(
+	<Alert>Alert content.</Alert>
+	, mountNode
+);
+```
+:::
+
+:::demo The conent can be non-strings such as a DOM element.
+```jsx
+import { Alert, Icon } from 'zent';
+
+ReactDOM.render(
+	<Alert>
+		<Icon type="error-circle" />
+		<span>Warning：SMS notification during the transaction will be sent via the "Message Push" function of the Marketing Center.</span>
```

Comment:
SMS notification of trading will be sent via the "Message Push" function of the Marketing Center.